src/views/admin/KlasseCreationView.vue

```diff
@@ -26,127 +20,40 @@
     );
-    return schuleIdsWithKlasseVerwaltenPermission.length === 1 ? schuleIdsWithKlasseVerwaltenPermission[0]! : null;
+    if (schule) return getDisplayNameForOrg(schule);
+    return '';
   });
-  const hasAutoselectedSchule: ComputedRef<boolean> = computed(() => autoselectedSchuleId.value !== null);
-
-  const validationSchema: TypedSchema = getValidationSchema(t);
 
-  const vuetifyConfig = (state: {
-    errors: Array<string>;
-  }): { props: { error: boolean; 'error-messages': Array<string> } } => getVuetifyConfig(state);
-
-  type KlasseCreationForm = {
-    selectedSchule: string | undefined;
-    selectedKlassenname: string;
-  };
-
-  // eslint-disable-next-line @typescript-eslint/typedef
-  const { defineField, handleSubmit, isFieldDirty, resetForm } = useForm<KlasseCreationForm>({
-    validationSchema,
+  const isFormDirty: Ref<boolean> = ref(false);
+  const hasUnsavedChanges: ComputedRef<boolean> = computed(() => {
+    if (organisationStore.createdKlasse) return false;
+    return isFormDirty.value;
   });
-
-  const [selectedSchule, selectedSchuleProps]: [
-    Ref<string | undefined>,
-    Ref<BaseFieldProps & { error: boolean; 'error-messages': Array<string> }>,
-  ] = defineField('selectedSchule', vuetifyConfig);
-  const [selectedKlassenname, selectedKlassennameProps]: [
-    Ref<string>,
-    Ref<BaseFieldProps & { error: boolean; 'error-messages': Array<string> }>,
-  ] = defineField('selectedKlassenname', vuetifyConfig);
-
-  const filteredSchulen: ComputedRef<TranslatedObject[] | undefined> = computed(() => {
-    return organisationStore.filteredSchulen.schulen
-      .slice(0, 25)
-      .map((org: Organisation) => ({
-        value: org.id,
-        title: org.kennung ? `${org.kennung} (${org.name.trim()})` : org.name,
-      }))
-      .sort((a: TranslatedObject, b: TranslatedObject) => a.title.localeCompare(b.title));
-  });
-
-  watch(
-    autoselectedSchuleId,
-    (newAutoSelectedOrgId: string | null) => {
-      const newSelectedSchule: TranslatedObject | undefined = filteredSchulen.value?.find(
-        (schule: TranslatedObject) => schule.value === newAutoSelectedOrgId,
-      );
-      selectedSchule.value = newSelectedSchule?.value;
-    },
-    { immediate: true },
-  );
-
-  const translatedSchulname: ComputedRef<string> = computed(
-    () =>
-      filteredSchulen.value?.find(
-        (schule: TranslatedObject) => schule.value === organisationStore.createdKlasse?.administriertVon,
-      )?.title || '',
-  );
-
-  function isFormDirty(): boolean {
-    const schuleDirty: boolean = hasAutoselectedSchule.value ? false : isFieldDirty('selectedSchule');
-    return schuleDirty || isFieldDirty('selectedKlassenname');
-  }
   const showUnsavedChangesDialog: Ref<boolean> = ref(false);
   let blockedNext: () => void = () => {};
 
+  function resetForm(): void {
+    // eslint-disable-next-line @typescript-eslint/dot-notation
+    formRef['value']?.reset();
+  }
```

Comment:
```suggestion
  function resetForm(): void {
     formRef.value?.reset();}
```
Dot Notation > Bracket Notation

Reply:
Eslint und Typescript widersprechen sich hier und ich habe die Typescript-Variante genommen. Ich kann es andersherum machen, aber ich glaube dann brauche ich ein ts-ignore.

Reply:
![image](https://github.com/user-attachments/assets/ed329a0f-f22b-4087-9c96-1399f822e509)
 
Bei mir geht es ohne Probleme. Vielleicht liegt es an deine IDE?

Reply:
Jap, jetzt hat er es genommen :')

Fixed in d8da458ad346f3102105080dc3ff51f7361d2786